doc/text/news.md

```diff
@@ -1,5 +1,40 @@
 # NEWS
 
+## 6.0.0: 2016-03-06 {#version-6-0-0}
+
+### Improvements
+
+  * Supported Groonga 6.0.0. Groonga 5.1.2 or older aren't supported.
+  * Supported `-` operator as {Groonga::Operator::AND_NOT}.
+  * `grndump`: Added `:max_records` option.
+    [Patch by Genki Takiuchi]
+  * Accepted integer and time as function call arguments.
+  * `expression-builder`: Disabled auto ID detection.
+    It's confused when Int/Uint types for key.
+  * Suported `require "rroonga"` for `Bundler.require`.
+    [Patch by Fumiaki MATSUSHIMA]
+  * Added {Groonga::Plugin.names}.
+  * Added {Groonga::Config#delete}.
+  * Added {Groonga::Config#each}.
+  * Added {Groonga::Index}. It has index column and section.
+  * Added {Groonga::Column#find_indexes}.
+    It returns an array of {Groonga::Index}.
+  * Deprecated {Groonga::Column#indexes}.
```

Comment:
Deprecated -> Mark as deprecated

`Deprecate` means protest against. 
`Deprecated` means to be planned dropping support. But `deprecated` is not *verb* but adjective.